tests/store/test_sqlalchemy_store.py

```diff
@@ -429,8 +435,11 @@ def test_log_metric(self):
         # MLflow RunData contains only the last reported values for metrics.
         with self.store.ManagedSessionMaker() as session:
             sql_run_metrics = self.store._get_run(session, run.info.run_id).metrics
-            self.assertEqual(2, len(sql_run_metrics))
-            self.assertEqual(1, len(run.data.metrics))
+            self.assertEqual(5, len(sql_run_metrics))
+            self.assertEqual(4, len(run.data.metrics))
+            self.assertTrue(math.isnan(run.data.metrics["NaN"]))
+            self.assertTrue(run.data.metrics["PosInf"] > 1e308)
```

Comment:
Can we assert it equals the value that we want exactlY?

Reply:
yeah makes sense